flutter_application_1/.metadata

```diff
@@ -0,0 +1,45 @@
+# This file tracks properties of this Flutter project.
```

Comment:
Don't push meta data files (.metadata, .gitignore) to git repo if the main code does not require them for running.

Reply:
did the required changes as requested.